main.nf

```diff
@@ -33,6 +33,8 @@ def helpMessage() {
       --mirna_gtf                   GFF/GTF file with coordinates positions of precursor and miRNAs. See: ftp://mirbase.org/pub/mirbase/CURRENT/genomes/hsa.gff3
       --bt_index                    Path to the bowtie 1 index files of the host reference genome. Optional.
       --mirtrace_species            Species for miRTrace. Pre-defined when '--genome' is specified. (hsa, mmu ...)
+      --refgenome                   Path to the Fasta file of the reference genome.
```

Comment:
What is the difference between the 'default' nf-core template `--fasta` and `--refgenome`?

Reply:
Okay, I think I should put the `--fasta` instead of the `--refgenome,` I didn't change the `--fasta ` in case it was being used by any process, but I think that is not the case. I will change the `--fasta` to `${params.igenomes_base}/Glycine_max/Ensembl/Gm01/Sequence/WholeGenomeFasta/genome.fa`